### PR TITLE
add ignorePath options

### PR DESCRIPTION
**Pull request by dcboy**
add ignorePath options
add test ignorePath logic

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "koa-bunyan-logger",
+  "name": "yp-koa-bunyan-logger",
```
Comment (oleg-koval) [inline package.json]: package name should remain the same

File: package.json [hunk 2]
```diff
@@ -17,13 +17,12 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/koajs/bunyan-logger.git"
+    "url": "https://github.com/dcboy/bunyan-logger.git"
```
Comment (oleg-koval) [inline package.json]: repo link should remain the same

File: package.json [hunk 3]
```diff
@@ -17,13 +17,12 @@
...
   },
   "bugs": {
-    "url": "https://github.com/koajs/bunyan-logger/issues"
+    "url": "https://github.com/dcboy/bunyan-logger/issues"
   },
-  "homepage": "https://github.com/koajs/bunyan-logger",
-  "author": "Pebble Technology <webteam@getpebble.com>",
+  "homepage": "https://github.com/dcboy/bunyan-logger",
```
Comment (oleg-koval) [inline package.json]: homepage link should remain the same

File: package.json [hunk 4]
```diff
@@ -17,13 +17,12 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/koajs/bunyan-logger.git"
+    "url": "https://github.com/dcboy/bunyan-logger.git"
   },
   "bugs": {
-    "url": "https://github.com/koajs/bunyan-logger/issues"
+    "url": "https://github.com/dcboy/bunyan-logger/issues"
```
Comment (oleg-koval) [inline package.json]: bugs link should remain the same

File: package.json [hunk 5]
```diff
@@ -39,4 +38,4 @@
     "mocha": "~2.3.3",
     "supertest": "~1.1.0"
   }
-}
+}
```
Comment (oleg-koval) [inline package.json]: add newline

**Comment by oleg-koval** [comment]
@sjmcdowall 
Don't you mind if we will put this PR to progress with fixing my comments, as the contributor is not responding, but this is a cool feature to add

**Comment by sjmcdowall** [comment]
@oleg-koval -- Not sure what you mean.  You just commented yesterday on the PR .. I agree it's a cool feature but your comments are spot on, this can't be merged as it is obviously even though the fixes are minor.  What is your suggestion here? How do we proceed? Someone grab the fork and fix the issues ??

**Comment by oleg-koval** [comment]
@sjmcdowall 
Yeah, this cant be merged, but I reviewed this PR some time ago, I think I can fork it and fix all comments for this feature

**Comment by oleg-koval** [comment]
Ill keep this PR open until we will have a new one

**Comment by oleg-koval** [comment]
# Motivation

> Use case: given a server with a `/ping` route a load balancer hits as a mechanism to determine availability. Now log lines for `/ping` are being produced every ~1s. I don't really want to log `/ping` at all


fixes #18
closes #21

# Changes

- add ignorePath options
- add test ignorePath logic

____
this PR is based on https://github.com/koajs/bunyan-logger/pull/21
thanks to @dcboy for contribution

**Comment by sjmcdowall** [comment]
So @oleg-koval -- This PR is for the dead 2X branch .. how do we go about and get this into merging into Master?

**Comment by oleg-koval** [comment]
@sjmcdowall true that, ill change da base

**Comment by oleg-koval** [comment]
@sjmcdowall 
so the base is changed, the branch is rebased to master as well

**Comment by sjmcdowall** [comment]
I don't think we should release a new version though until we fix up the deps, etc. and other clean up stuff.. but this is merged.

**Pull request merged**